.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,14 @@
+name: End to End Testing
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  e2e:
+    uses: upbound/uptest/.github/workflows/pr-comment-trigger.yml@main
+    with:
+      package-type: configuration
+    secrets:
+      UPTEST_CLOUD_CREDENTIALS: ${{ secrets.UPTEST_CLOUD_CREDENTIALS }}
```

Comment:
We need Support for Azure and AWS - think you can't use the generic e2e template here - in caas we have a version for multicloud

Reply:
Thanks for the great catch 👍 I switched to combined cloud credential format in https://github.com/upbound/configuration-dbaas/pull/1/commits/60c93b0e429ef8f5172f28107ccf1fef76caa7b2 to comply with the reusable workflow. This way we can avoid GHA workflow duplication and reuse the standard uptest one. 

```
--- PASS: kuttl (1224.85s)
    --- PASS: kuttl/harness (0.00s)
        --- PASS: kuttl/harness/case (1224.35s)
PASS
12:01:21 [ OK ] running automated tests
```